_layouts/default.html

```diff
@@ -49,31 +49,32 @@
         <a href="/"><img class="{{ logo_class }}" src="/assets/compiler_logo_stacked.svg" alt="Compiler logo"></a>
     </header>
 
-    <main>
+    <main class="container-fluid">
         {{ content }}
     </main>
 
-    <footer>
-        <div class="divider-line"></div>
-        <div class="certs d-flex flex-column flex-lg-row row-cols-lg-4">
-            <div>
-                <a href="mailto:hello@compiler.la" class="monospace contact-link">hello@compiler.la</a>
-                <p class="address">
-                    525 S Hewitt St<br />Los Angeles, CA 90013
-                </p>
-            </div>
+    <footer class="border-top border-light">
```

Comment:
The way this `border-light` Bootstrap class is getting its color is from the CSS root variable declaration here:

```css
    --bs-light-rgb: 84, 84, 84; /* for the HR above Footer */
    --bs-light: #545454; /* brand/scales/gray/4 */
```

More on `border-X` classes (like `border-primary`, `border-secondary`): https://getbootstrap.com/docs/5.2/utilities/borders/#color 
This also gives us the class: `text-light`: https://getbootstrap.com/docs/5.2/utilities/colors/#colors